frontend/awx/administration/settings/AwxSettingsForm.tsx

```diff
@@ -168,6 +169,38 @@ export function AwxSettingsForm(props: {
 
 export function OptionActionsFormInput(props: { name: string; option: AwxSettingsOptionsAction }) {
   const option = props.option;
+
+  if (option.label.endsWith('Secret')) {
```

Comment:
I read this code quickly @prat98 would this label being translated?

Reply:
Hmm good question maybe I should just do `.contains(t('Secret'))`. That way I capture the word in the label regardless. Certain translations may update the order of the words.

Reply:
I do not remember seeing such thing before. In general we might need to create an object or other data structure with an id that is not selected to be translated.

Reply:
Hmm so what is your proposal?